src/rule_definition_tools.jl

```diff
@@ -185,10 +185,14 @@ function scalar_rrule_expr(__source__, f, call, setup_stmts, inputs, partials)
     # because this is a pull-back there is one per output of function
     Δs = _propagator_inputs(n_outputs)
 
+    # Make a projector for each argument
+    ps, setup_ps  = _make_projectors(call.args[2:end])
```

Comment:
Can we say 
```suggestion
    projs, setup_projs  = _make_projectors(call.args[2:end])
```

This code is hard enough without more single letter variables :joy:

Reply:
This doesn't make the top 5 of my complaints about reading the macro... but sure!